spec/support/features/in_person_helper.rb

```diff
@@ -179,8 +179,16 @@ def complete_all_in_person_proofing_steps(user = user_with_2fa, tmx_status = nil
                                             same_address_as_id: true)
     complete_prepare_step(user)
     complete_location_step(user)
-    complete_state_id_controller(user, same_address_as_id: same_address_as_id)
-    complete_address_step(user, same_address_as_id: same_address_as_id) unless same_address_as_id
+    complete_state_id_controller(
+      user,
+      same_address_as_id: same_address_as_id,
+    )
+    unless same_address_as_id
+      complete_address_step(
+        user,
+        same_address_as_id: same_address_as_id,
+      )
+    end
```

Comment:
can we undo this change .. my guess this is due to a linter setting on your IDE 🙏🏿